manifests/image-references

```diff
@@ -2,11 +2,11 @@ kind: ImageStream
 apiVersion: image.openshift.io/v1
 spec:
   tags:
-  - name: cluster-group-upgrades-operator-precache
+  - name: telco-5g-ran4-topology-aware-lifecycle-operator-precache-rhel8
```

Comment:
this should just be the repository name `topology-aware-lifecycle-operator-precache-rhel8`